Cargo.toml

```diff
@@ -1,3 +1,10 @@
+[workspace.package]
+authors = ["IOG"]
+edition = "2021"
+homepage = "https://iohk.io/"
+repository = "https://github.com/input-output-hk/partner-chains.git"
+version = "1.6.0"
```

Comment:
due we keep consistent version among all the crates, I decided to put it here as well

Reply:
After some consideration I'm ok with this. If we change versioning scheme and rules, then we'll modify it.